dark/reads.py

```diff
@@ -378,41 +414,65 @@ def propertyDetails(self):
         """
         return (PROPERTY_DETAILS.get(aa, NONE) for aa in self.sequence)
 
-    def ORFs(self):
+    def ORFs(self, openORFs):
```

Comment:
Should `openORFs` have a default value?

Reply:
So far the `ORFs` function is only being called by `extract-ORFs.py`. There, the value for openORFs is parsed, and has a default value `False`. But it could make sense to put in a default here as well, for future use. 
`def ORFs (self, openORFS = False)`